src/actions/pause/components.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+
+import { getSelectedFrame } from "../../selectors";
+
+import type { ThunkArgs } from "../types";
+
+function getChildren(evaulate: Function) {
+  return evaulate(`
+    if(this.hasOwnProperty('_reactInternalFiber')) {
+      if (!this._reactInternalFiber.child) {
+        []
+      } else {
+        this._reactInternalFiber.child.memoizedProps.children
+          .filter(Boolean)
+          .map(child => ({
+            name: (child.type && child.type.name) || "",
+            child,
+            class: child.type
+          }))
+      }
+    } else {
+      []
+    }
+  `);
+}
+
+function getAncestors(evaluate: Function) {
```

Comment:
😢 

All of this can break in any patch release. *Please* don't rely on these. We don't want to break Firefox debugger.

We are happy to talk about turning these into semi-public APIs you could use. Let's work together on this.

Reply:
Specifically I believe we mentioned the `console.reactStack()` proposal before. It's not hard to add to React.

Reply:
@gaearon it would be nice to wrap revisit `console.reactStack` and see what kind of API makes sense.